bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TitleImpl.java

```diff
@@ -76,6 +92,16 @@ private void initModel() {
                 heading = Utils.Heading.getHeading(currentStyle.get(PN_DESIGN_DEFAULT_TYPE, String.class));
             }
         }
+
+        if (StringUtils.isNotEmpty(linkURL)) {
+            linkURL = Utils.getURL(request, pageManager, linkURL);
+        } else {
+            linkURL = null;
+        }
+
+        if(currentStyle != null) {
```

Comment:
fussy comment about the code style, space `if (currentStyle`

I'll generally try not to make many more comments about code style, as we can always clean it up later, but just keep it in mind.

Reply:
I'll surely keep all these things in mind the next time I make a commit. Thanks a lot for all the suggestions.